packages/suite-native-atoms/src/Button/TileButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { Icon, IconName } from '@trezor/icons';
+
+type ActionIconButtonProps = {
+    iconName: IconName;
+    title: string;
+    onPress: () => void;
+    style?: NativeStyleObject;
+};
+
+const rectangleButtonStyle = prepareNativeStyle(utils => ({
+    height: 79,
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: utils.borders.radii.small,
+    backgroundColor: utils.colors.gray300,
+    flex: 1,
+}));
+
+export const TileButton = ({ iconName, title, onPress, style }: ActionIconButtonProps) => {
```

Comment:
I am think about renaming this component to just RectangleButton, I know that's much more dummy but it's like super obvious how it look like

Reply:
Yeah, i thought about this. But rectangle is a shape that base button also has so it doesn't really say what it is. Tile is pretty common and is something very similar to this type of button. I am happy with both but I think rectangle is not saying much about the component at all.